plugins/modules/aci_l2out.py

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Sudhakar Shet Kudtarkar (@kudtarkar1)
+# Copyright: (c) 2020, Shreyas Srish <ssrish@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: aci_l2out
+short_description: Manage Layer2 Out (L2Out) objects.
+description:
+- Manage Layer2 Out configuration on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+  l2out:
+    description:
+    - The name of outer layer2.
+    type: str
+    aliases: [ 'name' ]
+  description:
+    description:
+    - Description for the L2Out.
+    type: str
+  bd:
+    description:
+    - Name of the Bridge domain which is associted with the L2Out.
+    type: str
+  domain:
+    description:
+    - Name of the external L2 Domain that is being associated with L2Out.
+    type: str
+  vlan:
+    description:
+    - The VLAN which is being associated with the L2Out.
+    type: int
+  target_dscp:
+    description:
+    - Description of the target
```

Comment:
We need to fix this description. Trying to find a better explanation.

Reply:
touche